### ISSUE TITLE
✨ Talent event deadline changes event (WIP)

### ISSUE THREAD

**Issue by petertgiles** [GCTC-NTGC/gc-digital-talent#12303]
## ✅ Acceptance Criteria

- [ ] When closed_at is changed on an open event "close_date_changed_at" becomes true, notification fired to all submitters with draft nominations in event (separate issue)

## 🛑 Blockers

```[tasklist]
### Blocked By
```

**Comment by petertgiles** [comment]
Feature scoped out and never built
